crates/ruff_linter/resources/test/fixtures/flake8_boolean_trap/FBT.py

```diff
@@ -71,6 +71,8 @@ def used(do):
 bar.is_not(False)
 next(iter([]), False)
 sa.func.coalesce(tbl.c.valid, False)
+setVisible(true)
+set_visible(true)
```

Comment:
Is it intentional to use the identifier `true` here instead of the singleton keyword value `True`?

Reply:
Uhh no